jpsreport/general/ArgumentParser.cpp

```diff
@@ -102,11 +102,11 @@ void Logs()
     // first logs will go to stdout
     Log->Write("----\nJuPedSim - JPSreport\n");
     Log->Write("Current date   : %s", currentTime.c_str());
-    // Log->Write("Version        : %s", JPSREPORT_VERSION);
+    Log->Write("Version        : %s", JPSREPORT_VERSION);
     Log->Write("Compiler       : %s (%s)", true_cxx.c_str(), true_cxx_ver.c_str());
-    // Log->Write("Commit hash    : %s", GIT_COMMIT_HASH);
-    // Log->Write("Commit date    : %s", GIT_COMMIT_DATE);
-    // Log->Write("Branch         : %s", GIT_BRANCH);
+    Log->Write("Commit hash    : %s", GIT_COMMIT_HASH);
+    Log->Write("Commit date    : %s", GIT_COMMIT_DATE);
+    Log->Write("Branch         : %s", GIT_BRANCH);
     Log->Write("Python         : %s (%s)\n----\n", PYTHON, PYTHON_VERSION);
```

Comment:
This output is no longer required. We removed the option to plot timeseries and Voronoi diagrams with python.

Reply:
I can remove it but it does not really have anything to do with this PR since I didn't change the line

Reply:
Okay. I remember it for a future pull request.

Reply:
@gjaeger I will fix this when cleaning up the cmakelists. See #574